followed-accounts.js

```diff
@@ -0,0 +1,56 @@
+import { db } from "./dbInstance.js";
+
+function formatDate(dateString) {
+  const options = {
+    year: "numeric",
+    month: "long",
+    day: "numeric",
+    hour: "2-digit",
+    minute: "2-digit",
+    second: "2-digit",
+    timeZoneName: "short",
+  };
+  const date = new Date(dateString);
+  return date.toLocaleDateString("en-US", options);
+}
+
+async function displayFollowedActors() {
+  const followedListElement = document.getElementById("followedList");
+  const followedActors = await db.getFollowedActors();
+  console.log(followedActors);
+
+  followedActors.forEach((actor) => {
+    const actorElement = document.createElement("div");
```

Comment:
We should have a component that displays the icon and name of an actor (reuse with the profile) and takes a url attribute. then we could create those elements here